sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventParticipantsGrid.java

```diff
@@ -80,7 +80,8 @@ public EventParticipantsGrid(EventParticipantCriteria criteria) {
 		caseIdColumn.setSortProperty(EventParticipantIndexDto.CASE_UUID);
 		caseIdColumn.setRenderer(new CaseUuidRenderer(uuid -> {
 			// '!=' check is ok because the converter returns the constant when no case creation is allowed
-			return NO_CASE_CREATE != uuid;
+
+			return NO_CASE_CREATE != uuid && FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.CASE_SURVEILANCE);
```

Comment:
Should check CASE_CREATE right too, please use `UiUtil.isPermitted`

Reply:
changed